src/Language/LexerInterface.php

```diff
@@ -26,8 +26,9 @@ public function lookahead(): Token;
     public function getBody(): string;
 
     /**
-     * @param string     $name
+     * @param string $name
      * @param mixed|null $default
+     *
```

Comment:
I haven't had an empty line here, but we could consider adding it to the code style included in the project.